src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -183,7 +183,16 @@
 
         <service id="sylius.api.section_resolver.shop_api_uri_based_section_resolver" class="Sylius\Bundle\ApiBundle\SectionResolver\ShopApiUriBasedSectionResolver">
             <argument>%sylius.security.new_api_shop_route%</argument>
+            <argument>%sylius.new_api_orders_resource_route%</argument>
```

Comment:
I wonder if defining this parameter will be useful, maybe you could pass the `orders` here
```suggestion
            <argument>orders</argument>
```